models/tasks.js

```diff
@@ -114,8 +113,12 @@ const fetchTask = async (taskId) => {
 
 const fetchUserTasks = async (username, statuses = []) => {
   try {
-    const { user } = await fetchUser({ username })
-    const userId = await userUtils.getUserId(user.username)
+    const userId = await userUtils.getUserId(username)
+
+    if (!userId) {
+      return false
+    }
```

Comment:
We shouldn't just return false here, we saw how it looks when `fetchUserTasks` is called, right?

We could instead return an object:
```JS
return { userNotFound: true }
```

Now at the time of function call, we can write:
```JS
if (allTasks.userNotFound) {
  return res.boom.notFound('User doesn\'t exist')
}
```


Does it feel better?